src/EditorFeatures/Core/ExternalAccess/IntelliCode/Api/IIntentSourceProvider.cs

```diff
@@ -77,21 +77,31 @@ public IntentRequestContext(string intentName, SnapshotSpan currentSnapshotSpan,
 
         /// <summary>
         /// The text changes that should be applied to the <see cref="IntentRequestContext.CurrentSnapshotSpan"/>
+        /// TODO - Remove once intellicode switches over to reading <see cref="DocumentChanges"/> instead.
         /// </summary>
+        [Obsolete("Use DocumentChanges instead")]
```

Comment:
rename requires us to support multiple document changes.  instead of outright removing this, obsoleting until intellicode can react to avoid dual insertion